src/codeu/chat/DerbyStore.java

```diff
@@ -137,7 +204,7 @@ public void updateLastMessage(Message m) throws SQLException {
 				// Add each created user to the store object
 				allUsers.insert(userid, user);
 			}
-			System.out.println("Successfully retrieved users");
+			LOG.info("Accessed users table.");
 		}
 		catch (Exception ex) {
 			ex.printStackTrace();
```

Comment:
Hmm I don't think I would.

Think about it this way: is the stack trace giving you any information that you really need? If you know the exception is going to happen, and you're using it as a way to detect that the tables need to be created, then you're probably never going to need to look at the stack trace to figure out exactly what's happening... right? A simple log saying "tables were not found, creating them now" or something would probably be enough, I would think.

Another way to think about it: try running the tests now. Right now I think the test log will contain this stack trace. That's going to make it look like something is failing, when in reality the stack trace is expected. It will also make it harder to figure out when something else fails, since you'll have to know to ignore this stack trace to find the real one.